doc/api/process.md

```diff
@@ -350,6 +350,9 @@ Signal events will be emitted when the Node.js process receives a signal. Please
 refer to signal(7) for a listing of standard POSIX signal names such as
 `SIGINT`, `SIGHUP`, etc.
 
+The signal handler will receive two arguments: the signal's name (`'SIGINT'`,
+ `'SIGTERM'`, etc.) and the signal's numeric code.
```

Comment:
Maybe move this below the next paragraph or merge them.  I think an unwitting reader would assume the second one is a continuation of the first, i.e., that it's about the callback's arguments.